libs/platform/user/ebpf_native_user.c

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "ebpf_result.h"
+#include "framework.h"
+
+ebpf_result_t
+ebpf_native_load_module(_In_z_ const wchar_t* service_name)
+{
+    UNREFERENCED_PARAMETER(service_name);
+    return EBPF_SUCCESS;
```

Comment:
should this return EBPF_OPERATION_NOT_SUPPORTED?

Reply:
Fixed.